crates/sargon-uniffi/src/wrapped_radix_engine_toolkit/low_level/manifest_summary/account_deposit/account_deposit.rs

```diff
@@ -1,9 +1,9 @@
 use crate::prelude::*;
-use sargon::AccountDeposit as InternalAccountDeposit;
+use sargon::AccountDeposits as InternalAccountDeposits;
 
 /// Represents an account deposit, which includes specified and unspecified resources.
 #[derive(Clone, PartialEq, Eq, InternalConversion, uniffi::Record)]
-pub struct AccountDeposit {
-    specified_resources: HashMap<ResourceAddress, SimpleResourceBounds>,
+pub struct AccountDeposits {
+    specified_resources: Vec<SimpleResourceBounds>,
     unspecified_resources: UnspecifiedResources,
```

Comment:
Now there is only one `unspecified_resources`, instead of having a potentially multiple for the same account.